main/include/imagewriter.h

```diff
@@ -0,0 +1,16 @@
+#ifndef IMAGEWRITER_H
+#define IMAGEWRITER_H
+#include <QDir>
+#include <memory>
+#include <QImage>
+
+class ImageWriter {
+public:
+    ImageWriter() { srand(time(NULL)); }
+    void save_to_directory(QDir path_to_, std::vector<std::shared_ptr<QImage>> images, QString format = ".jpg");
```

Comment:
```suggestion
    void save_to_directory(QDir path_to_, std::vector<std::shared_ptr<QImage>> images, QString format = ".jpg");
```
используй lowerCamelCase, пожалуйста, так как этот стиль больше соответствует QT + остальной код написан в нем